examples/quick-start-advanced/atmos.yaml

```
@@ -18,6 +18,16 @@
 base_path: "."
 
 components:
+  list:
```

Support a configurable default
```suggestion
  list:
    format: pretty
```

---

The default should be `pretty`